tests/golden/defaults/lieutenant/lieutenant/20_api/secret-lieutenant-api-user.yaml

```diff
@@ -1,5 +1,4 @@
 apiVersion: v1
-data: {}
```

Comment:
Ah right the kube library adds the empty `data` :facepalm:  I think we've had that issue somewhere before

Reply:
yeah, I'm sure we fixed this somewhere else, but I don't remember where.